doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -931,13 +934,14 @@ The enricher extracts the needed information from the camel-context.xml, and the
 The enricher looks for the `scheme`, `contextPath` and `apiContextPath` attributes, and it will add the following
 label and annotations:
 
-* The label `discovery.3scale.net/discoverable`. The value of the label can be set to "true" or "false", and was added to take part in the selector definition executed by 3scale to find all services that need discovery. Also it can act as a switch as well, to temporary turn off 3scale discovery integration by setting it to "false".
+* The label `discovery.3scale.net`. The value of the label can be set to "true" or "false", and was added to take part in the selector definition executed by 3scale to find all services that need discovery. Also it can act as a switch as well, to temporary turn off 3scale discovery integration by setting it to "false".
 
 * The following annotations:
-  - `discovery.3scale.net/discovery-version`: (optional) the version of the 3scale discovery process. By default its "v1"
-  - `discovery.3scale.net/scheme`: (optional) this can be "http" or "https". "http" by default.
-  - `discovery.3scale.net/path`: (optional) the contextPath of the service if it's not at the root.
-  - `discovery.3scale.net/description-path`: (optional) the path to the service description document (OpenAPI/Swagger). The path can either be relative or an external full URL. By default the description is picked up from the root context (/)
+  - `discovery.3scale.net/discovery-version`: the version of the 3scale discovery process. By default its "v1"
+  - `discovery.3scale.net/scheme`: this can be "http" or "https". "http" by default.
+  - `discovery.3scale.net/path`: the contextPath of the service if it's not at the root.
+  - `discovery.3scale.net/port`: the contextPath of the service if it's not at the root.
```

Comment:
why have you remove the optional part ? I found that quite useful.

Reply:
C&P Error: `/port` is *not* the "contextPath".

Reply:
I think everything is optional from a config perspective at this point as the code will always create the annotations that are required by 3scale. So to me it got confusing if it was a 'config required' or a '3scale required'.